packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -166,6 +168,28 @@ class GestureRecognizerFactoryWithHandlers<T extends GestureRecognizer> extends
 /// ```
 /// {@end-tool}
 ///
+/// {@tool sample}
+///
+/// This example of a red [Container] wraps a [GestureDetector] widget.
+/// Since the [GestureDetector] does not have a child it takes on the size of
+/// its parent making the entire area of the surrounding [Container] clickable.
+/// assumes the size of its parent, making the area of the surrounding [Container] clickable.
```

Comment:
The sentence starting with "assumes" is not really a sentence and it looks like its just repeating things from the previous sentence?